site/_posts/2021-04-08-open-source-auth-considerations.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog-post
+title: Open source auth provider considerations
```

Comment:
Maybe: the pitfalls of open source auth providers 
Or: the downsides of choosing open source auth providers

Reply:
maybe we don't want to be _so_ negative, but I think it sets better expectations than 'considerations'

Reply:
maybe: What to consider before choosing an open source auth provider

Reply:
`What to consider before choosing an open source auth provider` seems fair.